week-1/1-exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,20 @@
 function halve(number) {
   // complete the function here
-}
+return number /2;  
```

Comment:
To help make the code easier to read, lines written inside functions should be indented.

Reply:
i have worked on the feedback but it is showing that it is having conflict with the master repository

Reply:
Don't worry about the conflicts as these changes are not intended to be merged.  Great job on fixing the remainder function! 👍